reportutils/report.go

```diff
@@ -14,12 +15,26 @@ import (
 var httpClient = &http.Client{}
 
 // Report method requests the email service to send emails
-func Report(run *models.Run) {
+func Report(run *models.Run, receivers []string) {
 	common.LogInfo("Sending report...")
+
+	remark, ok := run.Settings[common.KeyRemark]
+	if ok && strings.ToLower(remark.(string)) == "official" {
+		for i := range receivers {
+			receivers[i] = fmt.Sprintf("%s@microsoft.com", receivers[i])
```

Comment:
This is a bit confusing. It is quite easy for the user to include full email addresses in the metadata.yml, in those case this will fail. Actually, I would rather enforce a full email name to be written in the metadata.yml so we do have to stick with `@microsoft.com` addresses.

Reply:
Well, I added this to avoid the spam

Reply:
Ok. So use secret / configmap instead if we don't want to expose email address?

Reply:
Ok. let's use alias to avoid spam then. I would prefer this also read from configmap and secret (this can be done later).